template/package-scripts.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Windows: Please do not use trailing comma as windows will fail with token error
+ */
+module.exports = {
+  scripts: {
+    default: 'webpack',
+    webpack: {
+      default: 'webpack.dev',
+      dev: 'webpack-dev-server --inline --progress --config build/webpack.dev.conf.js'
+    },
+    {{#if_or unit e2e}}
+    test: {
+      {{#if_eq runner "jest"}}
+      unit: 'jest --config test/unit/jest.conf.js --coverage'{{#e2e}},{{/e2e}}
```

Comment:
Why do we have a `e2e` flag in a unit test script?

Reply:
This flag adds trailing comma, if an `e2e` script is added on next line